test/testremotediscovery.cpp

```diff
@@ -63,7 +73,6 @@ private slots:
         // 200 should be an error since propfind should return 207
         QTest::newRow("200") << 200 << httpErrorMessage;
         QTest::newRow("InvalidXML") << +InvalidXML << "error while reading directory 'B' : Unknown error";
-        QTest::newRow("MissingPermissions") << +MissingPermissions << "error while reading directory 'B' : The server file discovery reply is missing data.";
```

Comment:
Eh, why do you remove this test?  Is there no way to keep it?
I guess, if you remove the test,  the `MissingPermissions` value should also go away.

Reply:
I'll remove the enum. I removed it from testRemoteDiscoveryError since it's no longer in the same class as the timeout or broken-xml error. It now has its own test.